test-harness/tests/open/test_master.py

```diff
@@ -46,8 +46,7 @@ def test_if_unknown_user_is_forbidden_access(self,
                                                 path_fmt.format(path),
                                                 )
 
-    @pytest.mark.parametrize("path", [(""),
```

Comment:
I did :)

We will still need it but in a different form. This test variant was checking how AR will react to bare `^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)(?<url>.*)$` URL, without anything added. The result should be that there are no extra slashes added. With the new endpoint being `^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)(?<type>(/logs/v1|/metrics/v0))(?<url>.*)$` we will need to test `/logs/v1` and `/metrics/v1` paths instead.

Let's leave AR tests for now and concentrate on integration tests. As soon as we confirm (or disprove) that the general idea works, we will freeze the working configuration in the unittests.

I hope it makes sense.